packages/dds/map/src/directory.ts

```diff
@@ -774,7 +820,7 @@ export class SharedDirectory
 	protected processCore(
 		message: ISequencedDocumentMessage,
 		local: boolean,
-		localOpMetadata: unknown,
+		localOpMetadata: DirectoryLocalOpMetadata,
```

Comment:
Since this might still be a remote op, can be undefined here.

```suggestion
		localOpMetadata: DirectoryLocalOpMetadata | undefined,
```